File: Bars.GkhGJI/libs/B4/model/Dict/erknm/RecordDirectoryERKNM.js
Ext.define('B4.model.dict.erknm.RecordDirectoryERKNM', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'RecordDirectoryERKNM'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name'},
        { name: 'Code'},
        { name: 'IdentSMEV'},
        { name: 'EntityId'},
        { name: 'CodeERKNM' },
        { name: 'IdentERKNM' },
        { name: 'DirectoryERKNM' },

    ]
});

File: Bars.GkhGJI/Entities/Dict/RecordDirectoryERKNM.cs
namespace Bars.GkhGji.Entities.Dict
{
    using Bars.Gkh.Entities;

    /// <summary>
    /// Запись справочники ЕРКНМ
    /// </summary>
    public class RecordDirectoryERKNM : BaseGkhEntity
    {
        public virtual string Name { get; set; }

        public virtual string Code { get; set; }

        public virtual string IdentSMEV { get; set; }

        public virtual long? EntityId { get; set; }

        public virtual string CodeERKNM { get; set; }

        public virtual string IdentERKNM { get; set; }

        public virtual DirectoryERKNM DirectoryERKNM { get; set; }
    }
}